src/Common/parseGlobs.cpp

```diff
@@ -116,4 +121,76 @@ std::string makeRegexpPatternFromGlobs(const std::string & initial_str_with_glob
     }
     return buf_final_processing.str();
 }
+
+namespace
+{
+void expandSelectorGlobImpl(const std::string & path, std::vector<std::string> & for_match_paths_expanded)
+{
+    /// regexp for {expr1,expr2,....};
+    /// expr1, expr2,... cannot contain any of these: '{', '}', ','
```

Comment:
Can exprN contain `*`? (if yes, please add to the list)

Reply:
it is a list of what cannot be in there :)
`*` can be there